packages/components/src/internal/components/search/FilterValueDisplay.tsx

```diff
@@ -67,13 +67,13 @@ export const FilterValueDisplay: FC<FilterValueDisplayProps> = memo(props => {
                         if (index > 5) return null;
                         if (index === 5) {
                             return (
-                                <li className="filter-display__filter-value-li">
+                                <li className="filter-display__filter-value-li" key={index}>
                                     <OverlayTrigger
                                         overlay={
                                             <Popover bsClass="popover" id={'filter-value-list-popover-' + index}>
                                                 <div>
-                                                    {[...values].splice(5).map(val => (
-                                                        <div>{val}</div>
+                                                    {[...values].splice(5).map((val, i) => (
+                                                        <div key={val}>{val}</div>
```

Comment:
Can use either val or i as key here, but since you explicitly add i, just checking if you intended to use val.

Reply:
Good catch. I had added `i` to use it for the key but then decided to just use `val` instead. Removing `i`.